### ISSUE TITLE
Location broadcast type thing

### ISSUE THREAD

**Issue by MattJeanes** [MattJeanes/TARDIS#65]
Some guy wanted a way to broadcast his location to everyone. Links with #59

**Comment by SingularMix** [comment]
You could have an option on the screen be something like a "TARDIS Locator" and it could have like a sonar scanner type thingy like a submarine.

**Comment by Noname12345GER** [comment]
The more often a TARDIS "Jumps" the better it can be found by this Scanner.

**Comment by jasontomasi** [comment]
Or, when a TARDIS is spawned, it could transmit its networked (GMOD XYZ) coordinates and made visible in all other TARDISes, perhaps with an option to select any of them and materialise nearby in the nearest suitable (wherever it fits) space?

**Comment by Noname12345GER** [comment]
Cmon Make Time-Ram possible :3

**Comment by MattJeanes** [comment]
Don't think this is worth doing. The linked idea in #59 for coordinate discs is a neat lore friendly way to share coordinates instead.

**Comment by parar020100** [comment]
I wanna give it a try, so I've reopened it for a while

**Comment by jasontomasi** [comment]
> Cmon Make Time-Ram possible :3

Indeed. I made a time-ram function in Wiremod's Expression2 but, it would be nice to have it natively, callable with E2 hooks. Given the variations in TARDIS interior designs, it would be nice if each author would conform to some minimal standard in terms of allowing space for a time-ram.
